packages/components/src/components/Card/Card.tsx

```diff
@@ -1,16 +1,42 @@
-import React, { ComponentProps } from "react";
-import cx from "classnames";
-
-type CardProps = ComponentProps<"div">;
-
-const CLASS_ROOT = "card";
-
-const Card = ({ className, ...other }: CardProps) => {
-  const classes = cx(CLASS_ROOT, className);
-
-  return <div className={classes} {...other} />;
-};
-
-Card.displayName = "Card";
-
-export default Card;
+import React from "react";
+import cx from "classnames";
+
+type Card = {
+  /** Specifies if content can overflow the card boundaries */
+  hasOverflow?: boolean;

```

Comment:
Toto plni nejaku funkciu, nastavujeme niekde nejaky overflow? Nenasiel som to nikde.